pkg/network/ebpf/c/protocols/http2/decoding.h

```diff
@@ -447,6 +447,19 @@ static __always_inline bool get_first_frame_header_without_remainder(pktbuf_t pk
     return format_http2_frame_header(current_frame);
 }
 
+static __always_inline bool get_first_frame_header_with_header_remainder(pktbuf_t pkt, frame_header_remainder_t *frame_state, http2_frame_t *current_frame) {
```

Comment:
documentation for the function

Reply:
I am a bit confused about the use of remainder and incomplete. You have changed a lot of places from remainder to incomplete but the function using remainder again?

Reply:
incomplete and remainder are not contradicting
anyway, renamed